client/app/Streamfinder.jsx

```diff
@@ -14,10 +14,30 @@ import MediaDetail from '../features/media/MediaDetail';
 import Account from '../features/accountPage/Account';
 import ErrorBoundary from '../features/sharedComponents/ErrorBoundary';
 import './Streamfinder.css';
-
+/* the idea:
+upon auth being valid - 
+* auth will send username and current history ids (from user schema) to StreamerFinder 
+  and state will bet set
+* Each component can use component did update for when their props
+  change.  upon this happening they can make an ajax request to server
+  to pull data needed from MovieSchema
+*/
 class Streamfinder extends React.Component {
   constructor(props) {
     super(props);
+
+    this.state = {
+      //user establish because this is prior to auth being hooked up
+      //this is under the impression auth was valid and currentId was sent
+      //to this component and updated via component did update.
+      currentId: 10138
```

Comment:
This is probably unnecessary. We'll be using localStorage to store user data, so you could just retrieve the username from there and set state in your own feature if you need to.

Reply:
yep, was just trying to get stuff working and local session wasnt set up yet.  So i did what i knew i could to make things work